src/components/utilities/requests.jsx

```diff
@@ -0,0 +1,16 @@
+import axios from "axios";
+
+const baseURL = axios.create({
+  baseURL: "https://nc-news-daniel.herokuapp.com/api",
+});
+
+export const fetchAllArticles = async (topic) => {
+  let endpoint = topic ? `/articles?topic=${topic}` : "/articles";
+  const { data } = await baseURL.get(endpoint);
+  return data.articles;
+};
```

Comment:
I'd look at changing this to use the axios params object. It will make it a lot easier when you come to doing the sort/order by